bodo/tests/test_parquet_write.py

```diff
@@ -36,8 +36,15 @@
 pytestmark = pytest.mark.parquet
 
 
-@pytest.fixture
-def check_write_func(datapath):
+def check_write_func(
```

Comment:
Modified testing function to no longer be a pytest fixture. Thats why there's all the green